json_serializable/test/src/_json_serializable_test_input.dart

```diff
@@ -120,6 +120,21 @@ class FinalFieldsNotSetInCtor {
   FinalFieldsNotSetInCtor();
 }
 
+@ShouldGenerate(r'''
+SetSupport _$SetSupportFromJson(Map<String, dynamic> json) {
+  return SetSupport((json['values'] as List)?.map((e) => e as int)?.toSet());
```

Comment:
I was thinking to modify first cast (`as List`) to `as Iterable` but realized that `dart:convert` returns list anyway

Reply:
yup